paasta_tools/cli/cmds/logs.py

```diff
@@ -169,6 +169,13 @@ def add_subparser(subparsers) -> None:
         "--lines +100 would give you the first 100 lines of logs. Defaults to the latest line's offset",
         type=int,
     )
+    status_parser.add_argument(
+        "-sh",
```

Comment:
please do not use two letters for a short option because that's very confusing. Choose one letter instead (capitals are fine, too)